Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -579,7 +580,9 @@
 
 void RemoteLayerTreeDrawingAreaProxy::hideContentUntilPendingUpdate()
 {
-    m_replyForUnhidingContent = sendWithAsyncReply(Messages::DrawingArea::DispatchAfterEnsuringDrawing(), [] { });
+    // FIXME(rdar://122365213): Rethink whether this needs to use an async reply handler or start a backgrounnd activity.
+    auto activity = m_webProcessProxy->throttler().backgroundActivity("hideContentUntilPendingUpdate"_s);
+    m_replyForUnhidingContent = m_webProcessProxy->sendWithAsyncReply(Messages::DrawingArea::DispatchAfterEnsuringDrawing(), [timedActivity = makeUnique<ProcessThrottlerTimedActivity>(1_s, WTFMove(activity))] () mutable { }, messageSenderDestinationID(), { }, WebProcessProxy::ShouldStartProcessThrottlerActivity::No);
```

Comment:
Had to use makeUnique here because otherwise it looks like the lambda was trying to initialize `timedActivity` via copy assignment.

Reply:
I would have thought that this would have worked:
```
auto activity = makeUnique<ProcessThrottlerTimedActivity>(1_s, m_webProcessProxy->throttler().backgroundActivity("hideContentUntilPendingUpdate"_s));
```
Then capture `activity = WTFMove(activity)` in the lambda?

What you did is fine though

Reply:
Yeah that works also. What didn't work was:

```
[timedActivity = ProcessThrottlerTimedActivity { 1_s, WTFMove(activity) }] () mutable { }
```

Presumably this fails because ProcessThrottlerTimedActivity is neither copyable nor movable.